docs/datamodel.rst

```diff
@@ -167,6 +167,24 @@ with information about the case - signalling that *this entity* belongs to *this
 the intention of the FMU results data model to maintain *all* information about a case, and 
 in the future it is expected that ERT will serve case information beyond the basics.
 
+    **Dot-annotation** - we like it and use it. This is what it means:
+
+    The metadata structure is a dictionary-like structure, e.g.
+
+    .. code-block:: json
+
+        {
+            "myfirstkey": {
+                "mykey": "myvalue",
+                "anotherkey": "anothervalue"
+                }
+        }
+    
+    ::
+
```

Comment:
This does not show(?) in HTML. De-indent perhaps?

Reply:
Shows when I build locally? 
![image](https://user-images.githubusercontent.com/35033325/157195638-f4552741-9f4e-4889-8776-9dd92c43d2e4.png)

Reply:
These intended blocks are meant as "info boxes" or whatever they are called. We could consider doing something on the styling side so that these get a different background or something to make them stand out. Currently, they are only intended a bit, but other than that look exactly the same as the rest of the text.